template/package.json

```diff
@@ -25,13 +25,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@emotion/native": "^10.0.6",
     "@emotion/core": "^10.0.6",
```

Comment:
adds `emotion-theming` and `styled-system`

Reply:
🔥